### PR TITLE
Complete basic Main Activity

### LINKED ISSUE #1: US 01.01.01
As an owner, I want to publish an experiment with a description, a region, and a minimum number of trials.

# Acceptance Criteria
- Have a button for creating a new experiment from the main activity.

- Button launches a fragment with the following fields:
    - Experiment description (TextEdit)
        - Unlimited length
    - A dropdown with Experiment types (Counts Binomial, Non-negative Count, Measurement)
    - Region (a text field to specify the general location where trials are expected to be conducted)
    - Minimum number of trials (used to indicate when an experiment can be ended)

- The fragment submits data for the experiment
    - If the data is valid, a new experiment is published and shown on the main activity.
    - Otherwise, an error message is shown.

- Have a storyboard demonstrating how a new experiment is created.
- Wiki entry
- UML for frontend components

- Link to relevant UML diagrams (#25) 

# Unknowns
- None

### PR DESCRIPTION
**Pull request by n30phyte**
Closes #1

Experiments can be added and the information for the experiments are shown on the main activity screen

### CODE REVIEW

File: app/src/main/java/xyz/kotlout/kotlout/view/TestUserActivity.java [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+  /**
+   * Upload user to firestore
+   *
+   * @param user
+   */
+  private void UpdateUser(User user) {
+    firestore.collection("users").document(user.getUuid()).set(user);
+  }
+
+}
```
Comment (AnmolDhaliwal00) [inline app/src/main/java/xyz/kotlout/kotlout/view/TestUserActivity.java:214]: This Activity should be deleted as it was used purely for experimenting.

**Review by AnmolDhaliwal00 — APPROVED** [review]
There's one leftover class from when we were experimenting with the user profiles, but other than that lgtm

**Review by PimpinPenguin — APPROVED** [review]
lgtm

**Review by Mia2b — APPROVED** [review]
lgtm!

**Pull request merged**